src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        List<String> strings;
+        int buy = 0;
+        int supply = 0;
+
+        try {
+            strings = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

This violates checklist item #11: 'Don't ignore exceptions'. When re-throwing an exception, you should provide a descriptive message to make debugging easier. For example: `throw new RuntimeException("Can't read data from file " + fromFileName, e);`. The same applies to the other `catch` blocks in this method.